aws/metadata/aurora/main.tf

```diff
@@ -0,0 +1,78 @@
+data "aws_vpc" "buf" {
+  id = var.vpc_id
+}
+
+resource "random_string" "aurora_password" {
```

Comment:
nit: Generally want to be using `random_password` for usecases like these instead of `random_string`, to make sure the output of this is marked as sensitive by default

https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password